default-fields/0000-default-fields.md

```diff
@@ -6,31 +6,206 @@
 # Summary
 [summary]: #summary
 
-One para explanation of the feature.
+Allow struct definitions to supply default values for individual fields,
+and then allow those fields to be omitted from struct initialisation:
+
+```rust
+struct Foo {
+    a: &'static str,
+    b: bool = true,
+    c: i32,
+    d: Vec<i32> = Vec::default()
+}
+
+let foo = Foo {
+    a: "Hello",
+    c: 42,
+}
+```
 
 # Motivation
 [motivation]: #motivation
 
-Why are we doing this? What use cases does it support? What is the expected outcome?
+Today, Rust allows you to create an instance of a struct using a literal syntax.
```

Comment:
I would start this section with a summary para which lists the motivations - privacy and boilerplate and that field defaults address this. Then go into detail about the two issue. ATM, the motivation reads like a few random issues and doesn't relate them to the proposal.